### ISSUE TITLE
Migrations - manage data structure within the database  

### ISSUE THREAD

**Issue by avelino** [nuveo/prest#53]
Migrations are pREST's way of propagating changes you make to your schema (adding a field, deleting a schema, etc.) into your database schema. They’re designed to be mostly automatic, but you’ll need to know when to make migrations, when to run them, and the common problems you might run into.

Vars:
```
PREST_MIGRATIONS
```

Commands:
```
# create new migration file in path
prest migrate create migration_file_xyz

# apply all available migrations
prest migrate up

# roll back all migrations
prest migrate down

# roll back the most recently applied migration, then run it again.
prest migrate redo

# show the current migration version
prest migrate version
```

**Comment by henriquechehad** [comment]
+1

**Comment by felipeweb** [comment]
Closed via #57
